app/app.go

```diff
@@ -232,20 +209,16 @@ type LinkApp struct { // nolint: golint
 	ParamsKeeper     paramskeeper.Keeper
 	AuthzKeeper      authzkeeper.Keeper
 	// IBC Keeper must be a pointer in the app, so we can SetRouter on it correctly
-	IBCKeeper        *ibckeeper.Keeper
-	TransferKeeper   ibctransferkeeper.Keeper
-	ICAHostKeeper    icahostkeeper.Keeper
 	EvidenceKeeper   evidencekeeper.Keeper
 	FeeGrantKeeper   feegrantkeeper.Keeper
 	ClassKeeper      classkeeper.Keeper
 	TokenKeeper      tokenkeeper.Keeper
 	CollectionKeeper collectionkeeper.Keeper
-	WasmKeeper       wasmpluskeeper.Keeper
+	RollupKeeper     rollupkeeper.Keeper
+	Ordakeeper       ordakeeper.Keeper
+	SettlementKeeper settlementkeeper.Keeper
 
-	ScopedIBCKeeper      capabilitykeeper.ScopedKeeper
-	ScopedTransferKeeper capabilitykeeper.ScopedKeeper
-	ScopedICAHostKeeper  capabilitykeeper.ScopedKeeper
-	ScopedWasmKeeper     capabilitykeeper.ScopedKeeper
+	ScopedICAHostKeeper capabilitykeeper.ScopedKeeper
```

Comment:
Since the other descriptions of ICA have been removed, would you like to do the same?

Reply:
Indeed, this field is already no longer used. I'll delete this.